yaas/parsers/service.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import inspect
+
+from .. import service
+from .. import config
+from .. import utils
+
+def define_subcommand(parent_parser):
```

Comment:
Okay, this is hella bike-shedding. I see you're point, but you could also just add the logic to your cli_args() function. If it is architecturally better later on, let's change it. Otherwise, let's get this PR in.